variables.tf

```diff
@@ -205,3 +205,32 @@ variable "minio_vault_secret" {
     vault_kv_secret_key_name = "secret_key"
   }
 }
+
+# Postgres
+variable "postgres_service" {
+  type = object({
+    service_name  = string,
+    port          = number,
+    username      = string,
+    password      = string,
+    database_name = string,
+  })
+  description = "Postgres data-object contains service_name, port, username, password and database_name"
+}
+variable "postgres_vault_secret" {
+  type = object({
+    use_vault_provider      = bool,
+    vault_kv_policy_name    = string,
+    vault_kv_path           = string,
+    vault_kv_field_username = string,
+    vault_kv_field_password = string
+  })
+  description = "Set of properties to be able to fetch Postgres secrets from vault"
+  default = {
+    use_vault_provider      = false
+    vault_kv_policy_name    = "kv-secret"
+    vault_kv_path           = "secret/data/dev/trino"
+    vault_kv_field_username = "username"
+    vault_kv_field_password = "password"
+  }
+}
```

Comment:
Voices in my head:
Have you thought about making the whole `postgres` as a source - optional feature for the `current terraform-nomad-trino` module? 

We might need to support several sources, and the user will decide which feature does he/she needs to turn on.

Reply:
I agree, that should be something we need to plan for. As of now, the issue addressed support for Postgres module only. But I think we need to find a way of making this dynamical to support any service (at least regarding connectors).

Reply:
Could you register an issue for optional use of sources?

Reply:
Done: #112